javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,78 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
```

Comment:
You can display the products using DOM manipulation instead of just console logging them.

Reply:
ok. I will do it